Kernel/FileSystem/VirtualFileSystem.cpp

```diff
@@ -56,7 +56,13 @@ ErrorOr<void> VirtualFileSystem::mount(FileSystem& fs, Custody& mount_point, int
             fs.class_name(),
             inode.identifier(),
             flags);
-        // FIXME: check that this is not already a mount point
+        const auto it = mounts.find_if([&mount_point](Mount const &existing_mount) {
```

Comment:
Please factor this out to a function we can share in both sides?

Reply:
Done - exported into a separate function.